src/components/TopBar/TopBar.test.js

```diff
@@ -24,17 +26,19 @@ afterEach(() => {
 it('renders', () => {
     act(() => {
         render(
-            <TopBar
-                appName="DemoApp"
-                onParametersClick={() => {}}
-                onLogoutClick={() => {}}
-                onLogoClick={() => {}}
-                user={null}
-            />,
+            <IntlProvider locale="en" messages={top_bar_en}>
+                <TopBar
+                    appName="DemoApp"
+                    onParametersClick={() => {}}
+                    onLogoutClick={() => {}}
+                    onLogoClick={() => {}}
+                    user={{ profile: { name: 'Username' } }}
+                />
+            </IntlProvider>,
             container
         );
     });
     expect(container.textContent).toBe(
-        '.powsybl_logo_svg__st0{fill:#ffb259}.powsybl_logo_svg__st1{fill:#6f2277}.powsybl_logo_svg__st2{fill:#fd495c}GridDemoApp'
+        '.powsybl_logo_svg__st0{fill:#ffb259}.powsybl_logo_svg__st1{fill:#6f2277}.powsybl_logo_svg__st2{fill:#fd495c}GridDemoAppUsername'
```

Comment:
why there's no space between GridDemoApp & Username  ?

Reply:
because the .textContent gives you only the text. in the dom, they are in different elements so they are not in a single word.